governance/roles.md

```diff
@@ -117,13 +124,27 @@ within the issue.
 
 Project Leads are nominated and approved by the following process:
 
-  1. Project Lead actively participates in the group, initiates the proposal or
-  volunteers to take on a project that has been prioritized by the group
-  1. A Chair or TL nominates a candidate, or requests the community to
-  nominate in a slack thread
-  1. The nomination is communicated via a pull request annotating the list
+1. Project Lead actively participates in the group, initiates the proposal or
+   volunteers to take on a project that has been prioritized by the group.
+2. A Chair or TL nominates a candidate, or requests the community to
+   nominate in a slack thread.
+3. The nomination is communicated via a pull request annotating the list
    of members in the [TAG README](/README.md) with a link to the issue tracking
    the project or on the issue tracking the project if a README is not yet made.
+   The nomination is typically open for a week (but may be shorter with LGTM
+   of at least two Chairs).
+4. The issue is updated with the assignment of the project lead as "assignee"
+   alongside the TAG Leadership member.
+5. Project Leads will be given the OWNER role of the directory or sub-directory
```

Comment:
is this technically possible? I don't believe you can do this in github today? Someone can clarify? @PushkarJ had a bunch of cool github gizmos he implemented through the bot though, so maybe that's possible?

Reply:
It's possible. This was a hindrance when I was the assessed rather than the assessor. We can accomplish so today by declaring ownership of directories or files in CODEOWNERS:  https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file